src/encoded/commands/deploy.py

```diff
@@ -1009,6 +1013,8 @@ def hostname(value):
     parser.add_argument('--cluster-size', type=int, default=5, help="Elasticsearch cluster size")
     parser.add_argument('--es-ip', default='localhost', help="ES Master ip address")
     parser.add_argument('--es-port', default='9201', help="ES Master ip port")
+    parser.add_argument('--primary-shards', default=3, type=int, help="ES number of primary shards")
+    parser.add_argument('--replicate-shards', default=1, type=int, help="ES number of replicate shards")
```

Comment:
Changing defaults from 5/2?

Reply:
yes, 5/2 was pushing us beyond the 1000 shard (primary and replicate) limit per cluster.